src/groups/special_euclidean.jl

```diff
@@ -21,7 +21,7 @@ SemidirectProductGroup(Tn, SOn, RotationAction(Tn, SOn))
 ```
 
 Points on $\mathrm{SE}(n)$ may be represented as points on the underlying product manifold
-$\mathrm{T}(n) \times \mathrm{SO}(n)$ or as affine matrices with size `(n + 1, n + 1)`.
```

Comment:
This use of `⨉` is also a little cramped.

Reply:
What should we do here then? It already has both spaces so I hoped it would render fine in both?